### PR TITLE
WT-12208 add application_cache_ops statistics and adjust application_evict_time's order

### PR DESCRIPTION
**Pull request by y123456yz**
add application_cache_ops statistics and Adjust application_evict_time's order

### CODE REVIEW

**Comment by y123456yz** [comment]
WT-12208

File: src/evict/evict_lru.c [hunk 1]
```diff
@@ -2521,6 +2522,7 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
     if (time_start != 0) {
         time_stop = __wt_clock(session);
         elapsed = WT_CLOCKDIFF_US(time_stop, time_start);
+        WT_STAT_CONN_INCR(session, application_cache_ops);
         WT_STAT_CONN_INCRV(session, application_cache_time, elapsed);
```
Comment (quchenhao) [inline src/evict/evict_lru.c:2526]: Reference for application_cache_time should be removed?

File: dist/stat_data.py [hunk 2]
```diff
@@ -740,8 +741,8 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Yield statistics
     ##########################################
+    YieldStat('application_cache_ops', 'application thread operations waiting for cache (usecs)'),
```
Comment (quchenhao) [inline dist/stat_data.py:744]: I think you mean the number of operations so the unit should not be usecs.

Comment (quchenhao) [reply dist/stat_data.py:744]: Can you describe the motivation of adding this stat?

**Comment by korteland** [comment]
@y123456yz would you be able to get back to @quchenhao 's questions please? I'll have to close this if I don't see any movement for a couple of weeks.

**Comment by y123456yz** [comment]
> @y123456yz would you be able to get back to @quchenhao 's questions please? I'll have to close this if I don't see any movement for a couple of weeks.

@korteland  I have change the code and add some descriptive information.
thanks.

**Comment by korteland** [comment]
@y123456yz Thanks - there are still some open questions in comment threads that need to be addressed though.

**Review by quchenhao — APPROVED** [review]
lgtm

**Comment by quchenhao** [comment]
@y123456yz Please resolve the conflicts.

**Comment by y123456yz** [comment]
@quchenhao 
I have merge develop and resolve the conflicts.
thanks.

**Review by quchenhao — APPROVED** [review]
lgtm

**Pull request merged**